src/ngAnimate/animateCss.js

```diff
@@ -623,6 +623,7 @@ var $AnimateCssProvider = ['$animateProvider', function($animateProvider) {
 
       if (options.delay != null) {
         var delayStyle = parseFloat(options.delay);
+        maxDelay = Math.max(options.delay, 0);
```

Comment:
Right, that looks more complex than I initially thought. I'll discuss this with Matias tomorrow.